src/programs/vault/transactions/UpdateExternalPriceAccount.ts

```diff
@@ -11,10 +11,7 @@ export class UpdateExternalPriceAccountArgs extends Borsh.Data<{
 }> {
   static readonly SCHEMA = new Map([
     ...ExternalPriceAccountData.SCHEMA,
-    ...this.struct([
-      ['instruction', 'u8'],
-      ['externalPriceAccount', ExternalPriceAccountData],
-    ]),
+    ...this.struct([['instruction', 'u8']]),
```

Comment:
Didn't work this way at all.

Reply:
Is it in sync with the latest metaplex foundation?

Reply:
Yes it does.